.github/workflows/stop_merges.yml

```diff
@@ -0,0 +1,14 @@
+name: DO NOT MERGE
+
+on:
+  pull_request:
+    types: [ labeled, opened, synchronize , reopened]
```

Comment:
I've added the "do not merge" label and the job failed as expected. but when i removed it, nothing changed.
Should we add event type "unlabeled" as well?